src/main/java/com/getjavaobj/training/algo/init/gzabarte/TaskCh02N013.java

```diff
@@ -0,0 +1,22 @@
+package com.getjavaobj.training.algo.init.gzabarte;
+
+public class TaskCh02N013 {
+    /**
+     * Reverse a number
+     *
+     * @param number integer to reverse
+     * @return number reversed
+     */
+    public static int reverseNumber(int number) {
+        String number_str = Integer.toString(number);
```

Comment:
As a rule, underscore is used only for constants. In other cases you need to use Uppercase, like this numberStr.